libafl_cc/build.rs

```diff
@@ -57,8 +57,6 @@ fn find_llvm_config() -> String {
             Ok(llvm_dir) => llvm_dir.to_str().unwrap().to_string(),
             Err(err) => {
                 println!("cargo:warning={}", err);
-                // falling back to system llvm-config
-                "llvm-config".to_string()
```

Comment:
We want to return the `brew` version if it was found

Reply:
Solved this one, I think. Now looking into the other workflows which are failing.